README.md

```diff
@@ -1,70 +1,112 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Covid Report
 
-## Available Scripts
+> "Covid Report" is a webApp designed for mobile. It is a Single Page App (SPA) that allows users to:
+> - To check the data on Covid-19.
+> - They can filter data by continent.
+> - They can access to each country data.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- [x] Languages: HTML, Sass and JavaScript
+- [x] Linters: Stylelint and ESLint
+- [x] Front-end library: React & Redux
+- [x] Code Editor: VS Code
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo 🔗
```

Comment:
* Kindly work on the live demo link as the details page on the live link isn't working as expected 
![image](https://user-images.githubusercontent.com/52497246/181135825-d5c1da82-6127-4661-82ba-282611e7b6ba.png)

Reply:
### @Davidosky007 

Hello. 

Here I asked a check from more than 4 end-users around the word and they haven't any issue with the App.
nobody reported any issue

![WhatsApp Image 2022-07-27 at 10 57 52 (2)](https://user-images.githubusercontent.com/40032163/181194298-49da9b76-c51f-4202-8f29-d333cb725c22.jpeg)
![WhatsApp Image 2022-07-27 at 10 57 52](https://user-images.githubusercontent.com/40032163/181194299-520daf2c-95b6-4a03-a619-e92a24f05cc7.jpeg)
![WhatsApp Image 2022-07-27 at 10 57 53](https://user-images.githubusercontent.com/40032163/181194300-40a594fa-5437-4aae-93d2-328aaa08651b.jpeg)
![WhatsApp Image 2022-07-27 at 10 57 52 (1)](https://user-images.githubusercontent.com/40032163/181194302-6f0eb176-5969-4fe1-bccd-5e0188c277ea.jpeg)